rcl/src/rcl/logging_rosout.c

```diff
@@ -33,41 +33,6 @@
 
 #define ROSOUT_TOPIC_NAME "/rosout"
 
-static rcl_ret_t rcl_ret_from_rcutils_ret(rcutils_ret_t rcutils_ret)
-{
-  rcl_ret_t rcl_ret_var;
-
-  if (RCUTILS_RET_OK != rcutils_ret) {
-    if (rcutils_error_is_set()) {
-      RCL_SET_ERROR_MSG(rcutils_get_error_string().str);
-    } else {
-      RCL_SET_ERROR_MSG_WITH_FORMAT_STRING("rcutils_ret_t code: %i", rcutils_ret);
-    }
-  }
```

Comment:
i am not sure why we need this. if needed, we can move the same logic into `rcl_convert_rcutils_ret_to_rcl_ret`?

Reply:
> To set the error message explictly if needed is a better way because we would like to keep the same consistent behavior in rcl.

i was wondering if this message is really useful for user. adding message here is mainly to add the `rcutils`'s return code in the error message. but that would not be useful for the user, since that does not tell why, and there should have been error message in the buffer in the previous `rcutils`'s function call. i am inclined to make it simple and fast to return the converted code to the user instead of that.